### PR TITLE
Py3 port

### PR DESCRIPTION
**Pull request by skrzepto**
Program works fine in py2 and py3 the problem is that unittests don't pass in py3 because of encoding problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,41 @@ fedmsg, the Fedora Federated Message bus.
 FMN.SSE allows fedora users to view their fedmsg feed in realtime.
 
 ## Install
+System dependencies
 ```
-sudo dnf install python python-virtualenvwrapper rabbitmq-server python-pip 
-mkvirtualenv sse
-workon sse
+sudo yum install python python-devel python-virtualenvwrapper rabbitmq-server python-pip gcc libffi-devel openssl-devel zeromq-devel
```
Comment (pypingou) [inline README.md]: You can split that over multiple lines with ``\`` if you like

**Comment by pypingou** [comment]
Looks fine to me but this encoding issue might be problematic no?

**Comment by skrzepto** [comment]
@pypingou Doesn't seem like it I ran python2 and python3 and both work fine in local hubs instance :)

**Comment by skrzepto** [comment]
Just the tests don't pass in python3 :/

**Comment by skrzepto** [comment]
@atelic fixed the unittests. So this could be good to merge

**Comment by pypingou** [comment]
If tests are passing :shipit: !

**Pull request merged**